### PR TITLE
docs: translate pricing page (part2)

### PR DESCRIPTION
**Pull request by d-bytebase**

### CODE REVIEW

File: locales/zh.yml [hunk 1]
```diff
@@ -10,3 +10,52 @@ pricing:
...
+        anomaly-detection-advanced: 高级
+        anomaly-detection-advanced-tooltip: 基础版功能+ Schema 偏差检测
+        schedule-change: 设定任务执行时间
+        review-and-backup-policy: 审核和备份策略
+        tenancy: 多地域/多租户部署
+    collaboration:
+      title: 团队协作
+      features:
+        ui-based-sql-review: 基于 UI 的 SQL 审查
+        vsc-workflow: "VCS 工作流 #GitOps"
```
Comment (boojack) [inline locales/zh.yml:47]:
```suggestion
        vsc-workflow: VCS 工作流
```

File: locales/zh.yml [hunk 2]
```diff
@@ -10,3 +10,52 @@ pricing:
...
+        schema-change: Schema 和数据变更
+        migration-history: 迁移历史
+        sql-editor: SQL 编辑器
+        database-backup-restore: 数据库备份和恢复
+        archiving: 归档
+        sql-check: SQL 检查
+        sql-check-basic: 基础
+        sql-check-basic-tooltip: 语法检查、数据库连接检查
+        sql-check-advanced: 高级
+        sql-check-advanced-tooltip: 基础版功能+向后兼容性检查
```
Comment (boojack) [inline locales/zh.yml:34]:
```suggestion
        sql-check-advanced-tooltip: 基础版功能 + 向后兼容性检查
```

File: locales/zh.yml (hunk 1 above)
Comment (ecmadao) [reply locales/zh.yml:47]: Maybe just align with the codebase? The translation there uses the "#GitOps"

Comment (boojack) [reply locales/zh.yml:47]: Then we don't need the `""`.
image

Comment (ecmadao) [reply locales/zh.yml:47]: The `""` is used to avoid yaml marks the `GitOps` as the comment:
图片
图片

File: locales/en.yml [hunk 3]
```diff
@@ -4,9 +4,56 @@ common:
...
+        anomaly-detection-advanced: Advanced
+        anomaly-detection-advanced-tooltip: Basic + Drift detection
+        schedule-change: Scheduled change at specific time
+        review-and-backup-policy: Review and backup policy
+        tenancy: Multi-Region / Multi-Tenancy deployment
+    collaboration:
+      title: Collaboration
+      features:
+        ui-based-sql-review: UI based SQL review
+        vsc-workflow: "VCS workflow #GitOps"
```
Comment (boojack) [inline locales/en.yml:45]:
```suggestion
        vsc-workflow: VCS workflow #GitOps
```

File: locales/zh.yml [hunk 4]
```diff
@@ -10,3 +10,52 @@ pricing:
...
+        anomaly-detection-advanced: 高级
+        anomaly-detection-advanced-tooltip: 基础版功能 + Schema 偏差检测
+        schedule-change: 设定任务执行时间
+        review-and-backup-policy: 审核和备份策略
+        tenancy: 多地域/多租户部署
+    collaboration:
+      title: 团队协作
+      features:
+        ui-based-sql-review: 基于 UI 的 SQL 审查
+        vsc-workflow: VCS 工作流 #GitOps
```
Comment (ecmadao) [inline locales/zh.yml:47]: I think we will only get the `VCS 工作流` if we don't have the `""`. The `GitOps` will be ignored as comment

**Review by boojack — DISMISSED** [review]
LGTM

File: locales/zh.yml (hunk 4 above)
Comment (boojack) [reply locales/zh.yml:47]: Yeah, it looks a little strange. If it doesn't show up in file, I will add it later.

Comment (boojack) [reply locales/zh.yml]: I will transform `.yml` to `.json` to fix the hot-reload in nuxt. So the `#GitOps` is ok in json file.

Comment (d-bytebase) [reply locales/zh.yml]: I changed to (GitOps). Either way works for me.

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**